### PR TITLE
notify when a new messageboard group is created

### PR DESCRIPTION
**Pull request by chrislewis60**
Just a little tweak to add a notify message when a messageboard group is created. There was even a message ready in the locales file but it just wasn't being called.

### CODE REVIEW

**Comment by chrislewis60** [comment]
Also just updated the message from the placeholder text 'Create a New Messageboard Group' to 'Messageboard group has been created'

**Comment by chrislewis60** [comment]
That Travis-CI fail does not look correct to me!

**Comment by chrislewis60** [comment]
The Travis CI failure is an example of the sporadic spec failures mentioned in #352 Sporadic spec failure in travis ci

File: config/locales/en.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ en:
       update: :thredded.form.update
       updated_notice: Messageboard has been updated
     messageboard_group:
-      create: Create a New Messageboard Group
+      create: Messageboard group has been created
```
Comment (glebm) [inline config/locales/en.yml]: This key is used in a few other places where this text would not make sense:

```
$ i18n-tasks find thredded.messageboard_group.create 
1 key matching 'thredded.messageboard_group.create' (4 usages) | i18n-tasks v0.9.5
thredded.messageboard_group.create 4
  app/views/thredded/messageboard_groups/new.html.erb:1 <% content_for :thredded_page_title, t('thredded.messageboard_group.create') %>
  app/views/thredded/messageboard_groups/new.html.erb:5 <li><%= t('thredded.messageboard_group.create') %></li>
  app/views/thredded/messageboard_groups/new.html.erb:17 <li><%= f.submit t('thredded.messageboard_group.create'), class: 'thredded--form--submit' %></li>
  app/views/thredded/messageboards/index.html.erb:18 <a class="thredded--button" href="<%= new_messageboard_group_path %>"><%= t('thredded.messageboard_group.create') %></a>
```

Please add a new `created_notice` key instead.

**Comment by chrislewis60** [comment]
Hi - I've reverted the Messageboard create notice and added a new created_notice instead. I've also added a Portuguese translation.

**Comment by chrislewis60** [comment]
Also just to mention that some of the other translations in the Portuguese file appear to have been changed. This occurred when I ran i18n-tasks add-missing to add the placeholder in the Portuguese file. I don't think it's materially changed anything - I assume it's just corrected some formatting issues in the file, but I could revert and only commit the created_notice line if you'd rather?

**Comment by jayroh** [comment]
I think there might have been some updates since you branched off master. If you rebase and push up your branch we can take a look at the diffs and see what needs adjusting

**Comment by chrislewis60** [comment]
Hi - I've rebased my forked master branch and pushed it back. https://github.com/chrislewis60/thredded - is that what you meant?

**Comment by glebm** [comment]
The difference in pt formatting are most likely due to a different (newer) version of the psych gem, no problem.

**Pull request merged**